autogpt_platform/frontend/src/app/globals.css

```diff
@@ -120,3 +120,12 @@
     @apply shadow-sm focus-visible:shadow-md;
   }
 }
+
+.scroll-none {
```

Comment:
seems not super needed but like who am I ya know

Reply:
also misleading because it doesn't make the thing non-scrollable like the name suggests (to me at least)